SDK Support Policy (#3319)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/sdk_support_policy.md

---
nav_title: SDK Support Policy
page_order: 0
---

# SDK Support Policy

This article describes Braze's SDK support policy and relevant timelines.

## Definitions

An SDK version's **end-of-life date** is the date at which Braze no longer supports that version. An SDK version's __end-of-life date__ will be 1 year after its next version is released.

> SDK versions which are **deprecated** are still fully supported, but have a known __end-of-life date__ in the future. For example, if Android SDK version 1.0.1 were released today, then for the next year its previous version (1.0.0) is considered deprecated, and will have an __end-of-life date__ one year from today.

Reviewer: 🤓 
```suggestion
SDK versions that are **deprecated** are still fully supported, but have a known __end-of-life date__ in the future. For example, if Android SDK version 1.0.1 were released today, then for the next year its previous version (1.0.0) is considered deprecated, and will have an __end-of-life date__ one year from today.
```

page_order: 0
---

# SDK Support Policy

This article describes Braze's SDK support policy and relevant timelines.

## Definitions

An SDK version's **end-of-life date** is the date at which Braze no longer supports that version. An SDK version's __end-of-life date__ will be 1 year after its next version is released.

SDK versions which are **deprecated** are still fully supported, but have a known __end-of-life date__ in the future. For example, if Android SDK version 1.0.1 were released today, then for the next year its previous version (1.0.0) is considered deprecated, and will have an __end-of-life date__ one year from today.

## SDK Support Timelines

Braze supports each version of an SDK for a minimum of one year. Once a newer version of an SDK is released, its previous version(s) will be considered __deprecated__, and will have an __end-of-life date__ in 12 months.

**Before** an SDK version reaches its __end-of-life date__, Braze will:

* Investigate bugs and release new SDK versions with enhancements.
* Answer questions related to this SDK version through usual support channels.

**After** an SDK version reaches its __end-of-life date__, Braze:

> * No longer guarantees successful message delivery for that older SDK version.

Author: ```suggestion
```

> * May not send messages in the case of known bugs for that older SDK version.

Author: ```suggestion
* May prevent messages being sent in the case of known bugs for that older SDK version.
```

> * Will not be able to provide support, troubleshoot integrations, or answer questions around that older SDK version.

Author: ```suggestion
* May not be able to provide support, troubleshoot integrations, or answer questions around that older SDK version.
```
